components/GitHubEventAction.tsx

```diff
@@ -86,19 +102,18 @@ function GitHubEventAction({type, payload}: GitHubEventActionProps): JSX.Element
       );
     }
     case 'PullRequestReviewEvent': {
-      const action = payload?.action;
-      const prNum = payload?.pull_request?.number;
-      const prURL = payload?.pull_request?.html_url;
+      const action = (payload as PullRequestReviewEvent).action;
+      const prNum = (payload as PullRequestReviewEvent).pull_request.number;
+      const prURL = (payload as PullRequestReviewEvent).pull_request.html_url;
       if (!action || !prNum || !prURL) {
         return unknown;
       }
-      const actionStr = action === 'created' ? 'reviewed' : action;
+      const actionStr = 'reviewed';
       return <span>{actionStr} <ELink link={prURL}>pull request #{prNum}</ELink> in</span>;
     }
     case 'PushEvent': {
-      const size = payload?.size; // should this be distinct_size?
```

Comment:
Looking at https://github.com/octokit/webhooks/blob/master/payload-types/schema.d.ts#L5052-L5095, probably number of items in `commits`?